evap/evaluation/tests/test_auth.py

```diff
@@ -38,3 +39,12 @@ def test_inactive_external_users_can_not_login(self):
         page = self.app.get(reverse("results:index") + "?loginkey=%s" % self.inactive_external_user.login_key).follow()
         self.assertContains(page, "Inactive users are not allowed to login")
         self.assertNotContains(page, "Logged in")
+
+    def test_login_key_resend_if_still_valid(self):
+        page = self.app.post("/", params={"submit_type": "new_key", "email": self.external_user.email}).follow()
+
+        new_key = UserProfile.objects.get(id=self.external_user.id).login_key
+        self.assertEqual(self.external_user.login_key, new_key)
```

Comment:
could you make it more explicit you're comparing the original key to the new one? like `old_key = ...; page = ...; new_key = ...; self.assertEqual(...)`